### ISSUE TITLE
Test wheelchair detection at AAF

### ISSUE THREAD

**Issue by cdondrup** [strands-project/strands_perception_people#197]
@lucasb-eyer @Pandoro we have until the 30/05 to test things on the robot at AAF before we pack up. If something breaks there is no harm to it any more so now is the time to test the wheelchair detector. Please reply here or on gitter if you want to do so, so we can arrange for that.

Might need instructions on how to run stuff ;)

**Comment by lucasb-eyer** [comment]
Thanks for the ping, I hope/think @Pandoro is doing it :smile: I unfortunately can't, you probably know why.

**Comment by cdondrup** [comment]
Yes, we started it Yesterday. He wasn't too impressed with the number of FP but we'll see.
